files/nrpe/check_nova_neutron_order.sh

```diff
@@ -21,25 +19,25 @@ do
     case $OPTION in
         h)
             usage
-            exit ${STATE_WARNING}
+            exit ${STATE_UNKNOWN}
             ;;
         \?)
             usage
-            exit ${STATE_WARNING}
+            exit ${STATE_UNKNOWN}
             ;;
         *)
             usage
-            exit ${STATE_WARNING}
+            exit ${STATE_UNKNOWN}
             ;;
     esac
 done
 shift "$((OPTIND-1))"
 
-IPTABLES_NEUTRON_STATUS="$(iptables --list --numeric|grep -E '^nova-filter-top|^neutron-filter-top'|head -1|grep neutron)"
+IPTABLES_NEUTRON_STATUS="$(sudo iptables --list --numeric|grep -E '^nova-filter-top|^neutron-filter-top'|head -1|grep neutron)"
 RETURNCODE=$?
 
 if [ "${RETURNCODE}" -ne 0 ]; then
-  echo "Nonzero $RETURNCODE returned from iptables ordering check."
+  echo "CRIT | Nonzero rc $RETURNCODE returned from iptables ordering check."
```

Comment:
Can we make this wording more obvious as to impact.
As sheriff : Why do I care about this check?
maybe
CRIT | customer firewall rules broken - ordering incorrect 
or something better worded.

Reply:
Sure, modified